components/navbar/navbar.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import Link from 'next/link'
+
+// Components
+import AvatarUser from '../svg/avatarUser'
+
+const Navbar = () => {
+    return (
+        <nav className="w-full flex items-center justify-between flex-wrap bg-blue-900 p-6" style={{ padding: "12px" }}>
+            <Link href="/" passHref>
+            <div className="flex items-center flex-shrink-0 text-white mr-6">
+                <h2>Bookmark</h2>
+            </div>
+            </Link>
+            <div className="block lg:hidden">
+                <button className="flex items-center px-3 py-2 border rounded text-teal-200 border-teal-400 hover:text-white hover:border-white">
+                    <svg className="fill-current h-3 w-3" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><title>Menu</title><path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z" /></svg>
+                </button>
+            </div>
+            <div className="w-full block lg:flex lg:flex-grow lg:items-center lg:w-auto">
```

Comment:
Yes, I agree with you, only I have this task on kanban `[FE 2]Create Search bar component` but I must do it in the next PR, but if you prefer I can delete this section.

Reply:
Ready, I created the component required, I'm having some difficulties with prop placeholder in consol because typescript doesn't accept props. Before this, in other components, I had errors similar

![Screen Shot 2020-11-11 at 20 03 11](https://user-images.githubusercontent.com/56928486/98886198-2e281480-2459-11eb-98e5-71c40afef688.png)

Also, I have been searching for info, but I don't sure 
-https://stackoverflow.com/questions/55075740/property-does-not-exist-on-type-intrinsicattributes
-https://www.pluralsight.com/guides/composing-react-components-with-typescript
-https://www.pluralsight.com/guides/use-interface-props-in-functional-components-using-typescript-with-react

If you prefer this part I think that would be a good try in the other task for search bar, issue [FE 2]Create Search bar component

Reply:
Hello, thank you for your help @jonathanperazawl I was a little confused with different solutions for typed typescript.

Ready!